freecad_api/main.py

```
@@ -198,140 +92,20 @@ async def process_grid(grid_data: GridData):
-        elif is_json_data:
-            temp_json = tempfile.NamedTemporaryFile(suffix=".json", delete=False)
-            with open(temp_json.name, "w") as f:
-                json.dump(grid_data, f)
-                
-            # 一時ファイルにFreeCADスクリプトを保存（モデル生成用）
-            temp_model_script = tempfile.NamedTemporaryFile(suffix=".py", delete=False)
-            with open(temp_model_script.name, "w") as f:
-                f.write(
-                    f"""
+        # 一時ファイルにFreeCADスクリプトを保存
+        temp_script = tempfile.NamedTemporaryFile(suffix=".py", delete=False)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)